### DISCUSSION TITLE
Pull Requests

### DISCUSSION THREAD

**Discussion by AbraaoAlves** [dcTeam23/course#6] [Q&A]
Olá futuros colegas de trabalho!
Nesse espaço espero compartilhar e responder algumas duvidas sobre como trabalhar com Pull Requests

Aqui vai algumas dicas de como fazer uma boa Pull Request:

1. Se ela está baseada em uma issue, coloque o mesmo nome da issue na PullRequest
2. Para vincular uma issue a uma PullRequest, basta escrever na descrição o seguinte conteudo: `Closes #<id-issue>` *
3. Lembre-se de criar sua PullRequest em modo de Draft p evitar feedbacks antecipados sobre seu trabalho que ainda não foi concluido
4. Ao concluir o trabalho na PR, basta clicar em `Ready to Review` na área inferior da PR
5. Também ao concluir o trabalho na PR, você deve marcar seu revisor na área de revisores (eu, e assim serei notificado que preciso avaliar seu trabalho) 

*: <id-issue> é o numero que fica logo após o titulo da issue.

Aqui um material [oficial do Github sobre PRs](https://docs.github.com/pt/pull-requests/collaborating-with-pull-requests), mas sintam-se a vontade de perguntar e responder sobre duvidas relacionadas a PullRequest aqui!

**Comment by AbraaoAlves** [comment]
Duvida do @breno-oliveira98 

![image](https://github.com/dcTeam23/course/assets/608731/4568b8bf-f295-4a44-90f6-c4ef0b6ad007)


# Reposta:

Use o [serviço de upload de imagem do proprio github](https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#uploading-assets)!

Aqui um exemplo:

https://github.com/dcTeam23/course/assets/608731/a8022d7c-b86f-43c6-bde3-8386165cf3d5


> E sim, esse video acima foi feito com a mesma tecnica!

![image](https://github.com/dcTeam23/course/assets/608731/233f0c99-e842-4491-b0aa-e0f381ce4c87)

**Reply by janiodslima** [reply]
é necessário fazer commit na README.md depois de fazer upload e criar o link?

**Reply by jmtannus** [reply]
Oi Janio, não é necessário. Use só pra gerar o link. Daí você recorta e cola no seu projeto.
